services/databases/_posts/2016-12-12-how-to-seed-your-database-from-a-dump-file.md

```diff
@@ -1,35 +1,43 @@
 ---
-title: How to seed your database from a dump file
-shortTitle: Seeding from a Dump File
+title: How To Seed Your Database From a Dump File
+shortTitle: Seeding From a Dump File
 step: 4
 tags:
 - services
 - databases
 - postgres
 - mysql
-description: How to seed your database from a dump file
+description: How To Seed Your Database From a Dump File
 ---
 
 One of the greatest features of Runnable is the ability to launch isolated application stacks for
 every branch that you commit. In fact, every single branch can have it's own database or set of
 databases to interact with, isolated from other branches, for testing any assortment of new
-features. As a result, breaking changes in your database schema or the addition of new models can
-be tested on the fly without affecting your _master_ branch databases.
+features.
 
-Our DB templates for Postgres and Mysql come with a build-in initialization script that allows you
-to seed you DB during the build process. This allows you to 'bake' your seed into your master DB
-container. Then every branch that you run in isolation will immediately start up with a DB
-container that is pre-seeded, taking advantage of the container image cache.
+As a result, breaking changes in your Database schema or the addition of new models can
+be tested on the fly without affecting your **master** branch databases.
 
-#### Get Started
+---
+
+Our Database templates for Postgres and Mysql come with a build-in initialization script that allows you
+to seed your Database during the build process. This allows you to 'bake' your seed into your master Database
+Container. Then every branch that you run in isolation will immediately start up with a Database
+Container that is pre-seeded, taking advantage of the container image cache.
+
+---
+
+### Getting Started
 In order to seed your database, you will need to [obtain a backup (or dump) of your existing database](clone_database.md).
 Once you have obtained the dump file, continue on the following steps:
 
 * [MySQL](#mysql)
 * [PostgreSQL](#postgresql)
 
-# MySQL
-### Upload the seed file to your template configuration
+---
+
+### MySQL
+Upload the seed file to your template configuration
 1. Bring up the template configuration modal by either clicking on the _cog_ next to your database template in the `Containers` pane, or clicking on Dockerfile in the `Templates` pane.
```

Comment:
You need a line break here or this will happen to you:

<img width="766" alt="screen shot 2017-04-20 at 6 55 09 pm" src="https://cloud.githubusercontent.com/assets/5341618/25259437/e6d817a2-25fa-11e7-8227-600fdcfad97b.png">